### DISCUSSION TITLE
how to add GIS layers? (BgMapWidget.tsx, MapLayersWidget.tsx)

### DISCUSSION THREAD

**Discussion by neuralmax** [iTwin/itwinjs-core#5465] [Q&A]
In the video "Visualizing GIS Data in Your iTwin Application"
https://www.youtube.com/watch?v=F_Ha1hGx8kU
There are BgMapWidget.tsx, MapLayersWidget.tsx files used and it is impossible to find them anywhere.
Is this video is outdated?
How to add GIS layers?

**Comment by pmconne** [comment]
@mdastous-bentley is the source code from this presentation available somewhere?

**Comment by mdastous-bentley** [comment]
This sandbox is quite close to what is being used in the video:
https://www.itwinjs.org/sandbox/MichelDastous/mapLayersExampleV1

If you have any question let me know.
